README.md

```diff
@@ -39,6 +44,10 @@ Now you can proceed to any of the following tasks:
 
 ## Cloning a Satellite host
 
+#### Prerequisites
+
+1. You will need files from a katello-backup (`katello-backup` on the `Satellite host`). The cloning playbook will work with or without the pulp_data.tar file from the backup.
```

Comment:
May be split this in to two lines like:

1. You will need files from a katello-backup (`katello-backup` on the `Satellite host`). 
    *Note:* The cloning playbook will work with or without the pulp_data.tar file from the backup.